projects/README.md

```diff
@@ -7,7 +7,7 @@
 |[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[patch](../patches/hdf5.patch 'patch')|
 |[Kerberos](http://web.mit.edu/kerberos/ 'Kerberos')|[open](http://web.mit.edu/kerberos/krb5-1.14/doc/mitK5license.html 'MIT Kerberos License information')|Kerberos is a network authentication protocol (only built on Windows)|4.0.1|none|none|
 |[librdkafka](https://github.com/edenhill/librdkafka 'librdkafka on github')|[open](https://github.com/edenhill/librdkafka/blob/master/LICENSE '2-clause BSD license')|librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support -- [windows-only patch](../patches/librdkafka-windows.patch)|v0.9.5|[repo](https://github.com/distributePro/librdkafka 'distributePro fork of librdkafka repo on github')|[diff](https://github.com/distributePro/librdkafka/compare/edenhill:v0.9.5...xp-v0.9.5 'patch/diff')|
-|[NOVAS](http://aa.usno.navy.mil/software/novas/novas_info.php 'NOVAS')|[open](http://aa.usno.navy.mil/software/novas/novas_c/README.txt '(See Section IV. Using NOVAS in Your Applications)')|NOVAS is an integrated package of ANSI C functions for computing many commonly needed quantities in positional astronomy.|3.1|none|[patch](../patches/novas.patch 'patch')|
+|[NOVAS](https://github.com/distributePro/NOVAS 'NOVAS')|[open](https://github.com/distributePro/NOVAS/blob/3.1/README.txt '(See Section IV. Using NOVAS in Your Applications)')|NOVAS is an integrated package of ANSI C functions for computing many commonly needed quantities in positional astronomy.|3.1|[repo](https://github.com/distributePro/NOVAS 'Mirror of the NOVAS C library')|none|
```

Comment:
I think I saw that you already created an issue for it, but this is showing that there is no patch for NOVAS, when there is in fact a patch being applied

Reply:
Yes, #79 . I noticed the issue while working on all this work. It beyond the scope of what we need at the moment, though.